src/test/acceptance/features/enter_name.feature

```diff
@@ -0,0 +1,10 @@
+Feature:
```

Comment:
Does the filename format need to be `enter_name.feature` rather than `enter-name.feature`?